packages/authorization-process/test/getClients.test.ts

```diff
@@ -0,0 +1,267 @@
+import {
+  Client,
+  PurposeId,
+  TenantId,
+  generateId,
+  unsafeBrandId,
+} from "pagopa-interop-models";
+import { describe, expect, it } from "vitest";
+import { getMockClient, getRandomAuthData } from "pagopa-interop-commons-test";
+import { genericLogger } from "pagopa-interop-commons";
+import { addOneClient, authorizationService } from "./utils.js";
+
+describe("getClients", async () => {
+  const consumerId = generateId();
+  const purposeId = generateId();
+  const mockClient1: Client = {
+    ...getMockClient(),
+    name: "test1",
+    consumerId: unsafeBrandId(consumerId),
+    kind: "Consumer",
+  };
+  const mockClient2: Client = {
+    ...getMockClient(),
+    name: "test2",
+    consumerId: unsafeBrandId(consumerId),
+    kind: "Consumer",
+  };
+
+  const userIds1 = generateId();
+  const userIds2 = generateId();
```

Comment:
```suggestion
  const userId1 = generateId();
  const userId2 = generateId();
```
Replace the others too, since it's a single Id it's confusing to use a plural